tests/sentry/web/frontend/test_organization_auth_settings.py

```diff
@@ -554,3 +556,82 @@ def test_edit_sso_settings__scim(self):
                 scim_role_restricted_user.flags["idp:role-restricted"],
             )
         )
+
+
+dummy_provider_config = {
+    "idp": {
+        "entity_id": "https://example.com/saml/metadata/1234",
+        "x509cert": "foo_x509_cert",
+        "sso_url": "http://example.com/sso_url",
+        "slo_url": "http://example.com/slo_url",
+    },
+    "attribute_mapping": {
+        Attributes.IDENTIFIER: "user_id",
+        Attributes.USER_EMAIL: "email",
+        Attributes.FIRST_NAME: "first_name",
+        Attributes.LAST_NAME: "last_name",
+    },
+}
+
+
+class DummySAML2Provider(GenericSAML2Provider):
```

Comment:
We have this identical class in `test_auth_saml2.py`, should we move it into a shared file so that if we need to change it, we only need to change it in one place?